packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -14,7 +15,7 @@
   display: flex;
   flex-direction: row;
   font-size: 12px;
```

Comment:
weird this wasn't a var

Reply:
We don't have a font value for 12px in tokens.